### ISSUE TITLE
Support streaming in `.Dir`

### ISSUE THREAD

**Issue by kana-rus** [ohkami-rs/ohkami#234]

**Comment by 1Cln** [comment]
Is this also related to dynamic files on the system not being mounted? As ohkami only recognizes the files that are present when it starts running.

**Comment by kanarus** [comment]
Yeah, static directory serving should be like a window that maps request paths to file paths in the directory, but current (wrong) implementation scans the directory once at startup and just hold files (and built handlers for them) on memory.
